src/js/view/controlbar.js

```diff
@@ -60,10 +60,9 @@ define([
 
 
     /** HTML5 Controlbar class * */
-    var Controlbar = function(_api, _config) {
-        _config = _config || {};
-        var _skin,
-            _dividerElement = _layoutElement('divider', CB_DIVIDER),
+    var Controlbar = function(_skin, _api, _model) {
+        _config = _model.componentConfig('controlbar') || {};
```

Comment:
```_config``` is now undeclared; global namespace pollution. Make sure you lint these kinds of errors before a commit and PR.